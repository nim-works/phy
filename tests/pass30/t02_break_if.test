(TypeDefs
  (ProcTy (Void)))
(GlobalDefs)
(ProcDefs
  (ProcDef (Type 0) (Params) (Locals)
    (Stmts
      (Drop (IntVal 100))
      (If (IntVal 0)
        (Stmts
          (Drop (IntVal 200))
          (Break 0)))
      (Drop (IntVal 300))
      (Return))))
