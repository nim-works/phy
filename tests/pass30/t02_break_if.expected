;$sexp
(TypeDefs
  (ProcTy (Void)))
(GlobalDefs)
(ProcDefs
  (ProcDef (Type 0) (Params) (Locals)
    (Stmts
      (Drop (IntVal 100))
      (SelectBool (IntVal 0)
        (Continue 1)
        (Continue 0))
      (Join 0)
      (Drop (IntVal 200))
      (Continue 1)
      (Join 1)
      (Drop (IntVal 300))
      (Return))))
